kyuubi-rest-client/pom.xml

```diff
@@ -33,13 +33,79 @@
 
     <properties>
         <jars.target.dir>${project.build.directory}</jars.target.dir>
+        <httpclient.version>4.5.13</httpclient.version>
```

Comment:
apache httpclient 4.x is in maintenance status, we can unify the version everywhere to use the latest 4.x